src/resources/postcss/views/full/month/_multiday-event.pcss

```diff
@@ -61,4 +61,10 @@
 		font-weight: var(--font-weight-regular);
 		text-overflow: ellipsis;
 	}
+
+	.tribe-theme-avada #main & {
```

Comment:
can we add a comment block above this like we do for the other theme overrides to clearly indicate where the specific theme override begins?

Reply:
Sure!